jbmc/src/jbmc/jbmc_parse_options.cpp

```diff
@@ -564,7 +566,7 @@ int jbmc_parse_optionst::doit()
     if(cmdline.isset("show-properties"))
     {
       show_properties(
-        goto_model, log.get_message_handler(), ui_message_handler.get_ui());
+        goto_model, log.get_message_handler(), message_handler.get_ui());
```

Comment:
🍌 Are `log.get_message_handler()` and `message_handler` actually different?

Reply:
I hope not. But I will check.

Reply:
Message_handler and ui shouldn't be passed separately.